src/assistant_library_demo.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A demo of the Google Assistant Library"""
+
+import sys
+
+import aiy.assistant.auth_helpers
+import aiy.voicehat
+
+
+def main():
+    """Run a recognizer using the Google Assistant Library.
+
+    The Google Assistant Library has direct access to the audio API, so this
+    Python code doesn't need to record audio.
+    Hot word detection "OK, Google" is supported.
+    """
+
+    try:
+        from google.assistant.library import Assistant
```

Comment:
Move this to the top. The only reason it was in the function before is so you can use the other backends without installing the Assistant Library, but this file doesn't support other backends, so it can be in the proper place.